soh/soh/resource/logging/SceneCommandLoggers.h

```diff
@@ -0,0 +1,31 @@
+#include "Resource.h"
```

Comment:
```suggestion
#include "Resource.h"
#include "soh/OTRGlobals.h"
```

Reply:
whoops, windows built fine so I thought the build fails were just the wii u stuff